readme.markdown

```diff
@@ -196,6 +196,25 @@ X bytes written (Y seconds)
 
 with the number of bytes in the bundle X and the time in seconds Y.
 
+# working with browserify transforms
+
+If your custom transform for browserify adds new files to bundle in a non-standard way without requiring.
+You can inform Watchify about these files by emiting 'file' event.
```

Comment:
Some small spelling improvements:

 - to bundle → to _the_ bundle
 - emiting 'file' event → emitting a 'file' event